packages/flutter_tools/test/general.shard/build_system/targets/macos_test.dart

```diff
@@ -2,22 +2,47 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_tools/src/base/build.dart';
 import 'package:flutter_tools/src/base/file_system.dart';
 import 'package:flutter_tools/src/base/io.dart';
 import 'package:flutter_tools/src/base/platform.dart';
 import 'package:flutter_tools/src/base/process_manager.dart';
 import 'package:flutter_tools/src/build_system/build_system.dart';
-import 'package:flutter_tools/src/build_system/exceptions.dart';
 import 'package:flutter_tools/src/build_system/targets/dart.dart';
 import 'package:flutter_tools/src/build_system/targets/macos.dart';
 import 'package:flutter_tools/src/cache.dart';
 import 'package:flutter_tools/src/macos/cocoapods.dart';
+import 'package:flutter_tools/src/macos/xcode.dart';
 import 'package:mockito/mockito.dart';
 import 'package:process/process.dart';
 
 import '../../../src/common.dart';
 import '../../../src/testbed.dart';
 
+const String _kInputPrefix = 'bin/cache/artifacts/engine/darwin-x64/FlutterMacOS.framework';
+const String _kOutputPrefix = 'macos/Flutter/ephemeral/FlutterMacOS.framework';
+
+final List<File> inputs = <File>[
+  fs.file('$_kInputPrefix/FlutterMacOS'),
+  // Headers
+  fs.file('$_kInputPrefix/Headers/FLEDartProject.h'),
+  fs.file('$_kInputPrefix/Headers/FLEEngine.h'),
+  fs.file('$_kInputPrefix/Headers/FLEViewController.h'),
```

Comment:
Same changes here, and the umbrella header also.

Reply:
List should be up to date, add the umbrella header back too